README.md

```diff
@@ -62,7 +65,11 @@ Note: You can compile the game with Mumble Positional Audio support, by defining
 ## Installing compiled binaries
 
 ### Windows
-When you click on the AppVeyor icon or go to https://ci.appveyor.com/project/magnus-gross/quickcurver directly, you can click on Artifacts on the latest build and download a zip file. Extract all files and run `QuickCurver.exe` in the `release` directory.
+When you click on the AppVeyor icon or go to https://ci.appveyor.com/project/magnus-gross/quickcurver directly, you can download and see the automated AppVeyor build.
+If you are not interested in the technology behind it and you finally want to play, just click here:
+https://ci.appveyor.com/api/buildjobs/j8ptwckm84f1tn23/artifacts/build%2FQuickCurver.zip
```

Comment:
Unfortunately, this won't work, as your link includes the build job id and thus will not update automatically once a new build job comes in (which is the case for every master push).

Reply:
True, but now you changed it by yourself to the artifact link?